docs/user/collected-metrics/metrics.md

```diff
@@ -35,14 +50,38 @@ The following metrics are added to the ping:
 | glean.baseline.locale |[string](https://mozilla.github.io/glean/book/user/metrics/string.html) |The locale of the application (e.g. "es-ES").  |[1](https://bugzilla.mozilla.org/show_bug.cgi?id=1512938#c3)||never |
 
 ## deletion-request
+
 This ping is submitted when a user opts out of sending technical and interaction data to Mozilla. This ping is intended to communicate to the Data Pipeline that the user wishes to have their reported Telemetry data deleted. As such it attempts to send itself at the moment the user opts out of data collection.
 
+
 This ping is sent if empty.
+
 This ping contains no metrics.
 
 ## metrics
+
 This is a built-in ping that is assembled out of the box by the Glean SDK.
+
 See the Glean SDK documentation for the [`metrics` ping](https://mozilla.github.io/glean/book/user/pings/metrics.html).
+
+Reasons this ping may be sent:
+    - `overdue`: The last ping wasn't submitted on the current calendar day, but it's after
```

Comment:
Oh, I think there's still a bug here: if you see the rendered version, you'll notice that we still need a newline before this otherwise `overdue` will get printed on the same line as "Reasons this ping may be sent"

Reply:
Good catch.  I'll have to go back to glean_parser and make a new release ... blablabla...